File: Keller.py
from hydle_family import keller

def me(name, age, qualities):
  name = 'Keller'
  age = 15
  qualities = {'funny': keller.funny(), 
               'kind': keller.kind(), 
               'smart': keller.smart()
              }

if __name__ == '__keller__':
  k = keller()
  k.me()

File: Keller.cpp
#include <iostream>
#include <vector>

using std::string, std::cout, std::vector;

class Keller
{
  public:
    string name;
    int age;
    vector<string> qualities;

  Keller(string name, int age, vector<string> qualities){
    this->name = name;
    this->age = age;
    this->qualities = qualities;
  }
};

int main()
{
  vector<string> qualities = {"Snowboarding", "Wakeboarding", "German Learning", "Programming", "Math"}; // TODO: add more qualities

  Keller keller("Keller Hydle", 15, qualities);
  
  return 0;
}
